bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/channelhelper/EventPersonChannelHelper.java

```diff
@@ -31,13 +31,15 @@
 @NonNullByDefault
 public class EventPersonChannelHelper extends EventChannelHelper {
     public EventPersonChannelHelper() {
+        // TODO : this would be a small breaking change but we should use the group extension mecanism
+        // person-event would become last-event, the same than other eventable things. Talk with @lolodomo before 3.3
```

Comment:
If it will make it into 3.3 breaking changes doesn't matter since already broken.

Reply:
Changed, also documentation.